Wrappers/Python/ccpi/viewer/utils/hdf5_io.py

```diff
@@ -115,7 +115,7 @@ def GetDimensions(self):
             return f[self.__DatasetName].shape[::-1]
 
     def GetOrigin(self):
-        print("Not yet implemented, set to (0,0,0) by default.")
+        # print("Not yet implemented, set to (0,0,0) by default.")
```

Comment:
this comment makes me think there's something not well working here?

Reply:
In a later PR we will be having a writer which writes out a HDF5 dataset (which we have resampled / cropped) to a hdf5 file along with the origin and other details, so then we could read the origin from there in future

Reply:
For docs on the geometry in Nexus files (which are a specification of format for HDF5) see for instance 
*  https://manual.nexusformat.org/design.html#design-fields
*  https://manual.nexusformat.org/classes/base_classes/NXgeometry.html#nxgeometry